features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -333,3 +333,23 @@
   step %{I open signups for "Awesome Gift Exchange"}
   step %{everyone has signed up for the gift exchange "Awesome Gift Exchange"}
 end
+
+### Deleting things
+# use the same names as "everyone is signed up"
+
+Then /^no one should be signed up for "([^\"]*)"$/ do |challenge|
+  collection = Collection.find_by_title(challenge)
+  %w(myname1 myname2 myname3 myname4).each do |name|
```

Comment:
I think it should check all users. It hurts the eye to see all users then a literal list :(